frontend/galeri/src/components/Header.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import "./Header.scss";
+
+export default class Header extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            qry: ""
```

Comment:
```suggestion
            query: ""
```

Unnecessary abbreviation.